rivus/utils/prerun.py

```diff
@@ -0,0 +1,45 @@
+"""
+    Collection of small rivus related util functions
+    
+    In use to avoid multiple solutions of the same
+    problem.
```

Comment:
This line-wrap looks suspiciously unmotivated. Also, why is this docstring indented?